packages/fluent-theme/src/components/preChatActivity/PreChatMessageActivity.module.css

```
@@ -0,0 +1,36 @@
+:global(.webchat-fluent) .pre-chat-message-activity {
+  display: grid;
+  grid-template-areas: 'body' 'toolbar';
+  grid-template-rows: auto auto;
+  gap: 32px;
+
+  padding: 32px; /* Unproven number. */
+}
+
+:global(.webchat-fluent) .pre-chat-message-activity__body {
+  font-family: var(--webchat-fontFamilyBase, 'Segoe UI');
+  font-size: 14px;
+  font-weight: 400;
+  grid-area: body;
+  line-height: 20px;
+  text-align: center;
+}
+
+:global(.webchat-fluent) .pre-chat-message-activity__body h2 {
+  color: var(--webchat-colorNeutralForeground1, #242424);
```

```suggestion
  color: var(--webchat-colorNeutralForeground1);
```
See previous comment